docroot/.htaccess

```diff
@@ -69,9 +69,10 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
-  # Forbid access to admin and node paths on the api2.foia.gov domain.
+  # Forbid access to admin, node, and user paths on domains other than admin.foia.gov and local.dojfoia.gov.
+  RewriteCond %{HTTP_HOST} !local.dojfoia.gov
   RewriteCond %{HTTP_HOST} !(.*)admin\.foia\.gov$ [NC]
-  RewriteRule ^(admin.*|node/.+) - [F]
+  RewriteRule ^(admin.*|node/.+|user.*) - [F]
```

Comment:
This will also block access to `/administration` and `/users`, which isn't the intention. Let's do something like this:
```
RewriteRule ^(admin($|/)|node($|/)|user($|/)) - [F]
```

Also, I just noticed we're not blocking `core/install.php` or `update.php`. We probably should, but that one wouldn't be restricted by domain. No one should really ever be accessing those. Let's do a rewrite rule before any of the rewrite condition statements:
```
RewriteRule ^(core/install\.php|update\.php) - [F]
```